tests/containers/cc/Makefile

```diff
@@ -11,14 +11,18 @@ CC=python-helloworld-cc
 CC_stamp=$(CC).stamp
 
 # This test verifies in-enclave networking.
-# Due to the use of the Azure DCAP Client on the host,
-# it also implicitly verifies host-side networking (not using the TAP device).
-CC_DOCKER_ARGS=\
-	--rm --privileged --network=host \
+
+CC_DOCKER_ARGS_COMMON=\
+	--rm \
+	--user $(shell id -u) \
+	--device /dev/net/tun --network host \
 	-v $(SGXLKL_PREFIX):/opt/sgx-lkl \
 	-e SGXLKL_VERBOSE=1 -e SGXLKL_KERNEL_VERBOSE=1 -e SGXLKL_TAP=sgxlkl_tap0 \
 	-e AZDCAP_DEBUG_LOG_LEVEL=INFO
 
+CC_DOCKER_ARGS_HW=\
+	--device /dev/sgx
```

Comment:
After creating a CC, how will a user learn about these parameters for docker run? Is this just documented in README.md? In addition, do we have `sgx-lkl-docker` or `sgx-lkl-cfg` output these after creating the CC?

Reply:
This is documented in `docs/Docker.md` and as mentioned in that PR there will be follow-up PRs to integrate this into the overall documentation.